src/netif.c

```diff
@@ -129,6 +129,65 @@ static struct list_head port_ntab[NETIF_PORT_TABLE_BUCKETS]; /* hashed by name *
  *       unregistered on cleanup stage
  */
 
+static inline void rte_pktmbuf_copy_metadata(struct rte_mbuf *mi, struct rte_mbuf *m)
```

Comment:
Pls add mbuf related API to `src/mbuf.c` and `include/mbuf.h`.
And do not start with `rte_xxxx`, because it is for DPDK library, you can use `mbuf_copy_metadata`, `mbuf_copy`， etc.